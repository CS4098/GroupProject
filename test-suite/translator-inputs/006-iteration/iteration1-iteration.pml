process iteProc {

    action a {}
    iteration {
        action b {}
        action c {}
    }
    action d {}
}
