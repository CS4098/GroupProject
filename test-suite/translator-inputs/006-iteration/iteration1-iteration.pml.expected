
active proctype iteProc()
{
	int count__;
    for (count__ : 1 .. 4) {
    }
}
